ui/src/message_popup/renderers/long_text_response.jsx

```diff
@@ -0,0 +1,111 @@
+/* @flow weak */
+import React from 'react';
+
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+
+
+// This response supports responding with text.
```

Comment:
Copy and pasting works well for certain components, and for others it's worth figuring out how to change them so that they can just be passed different parameters.  For the experience pages, those are the top-level component so we've been doing more copy-pasting that we might do for a smaller component that is re-used.

It's super helpful how you called out that this component is the same as `MinimalTextResponse` but just longer.  This would be a great case to change `MinimalTextResponse` so that the height could be passed in as a parameter, rather than having to copy and create a new component.  That way all the other details in here don't need to be repeated, and can stay defined in one place.

But it turns out that `MinimalTextResponse` already supports that - you can just pass in `textHeight: 192` or whatever you like.  So you can delete this component, and then update https://github.com/mit-teaching-systems-lab/threeflows/pull/281/files#diff-22c87e71dea0aa864896af1a42a86397R63 to call `MinimalTextResponse` but with a `textHeight` prop.